src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/GraphQLMiddleware.cs

```diff
@@ -184,8 +181,10 @@ private async Task WriteErrorAsync(HttpContext context, string message, Exceptio
                 throw new ArgumentNullException(nameof(message));
             }
 
-            var errorResult = new ExecutionResult();
-            errorResult.Errors = new ExecutionErrors();
+            var errorResult = new ExecutionResult
+            {
+                Errors = new ExecutionErrors()
```

Comment:
hmmm - do we never return errors?

Reply:
yes they are added after, in different ways depending if an exception was passed to this method.